src/Gui/DlgGeneral.ui

```diff
@@ -505,6 +505,43 @@ this according to your screen size or personal taste</string>
            </item>
           </layout>
          </item>
+         <item>
+          <layout class="QHBoxLayout">
+           <property name="spacing">
+            <number>6</number>
+           </property>
+           <property name="leftMargin">
+            <number>0</number>
+           </property>
+           <property name="topMargin">
+            <number>0</number>
+           </property>
+           <property name="rightMargin">
+            <number>0</number>
+           </property>
+           <property name="bottomMargin">
+            <number>0</number>
+           </property>
+           <item>
+            <widget class="QLabel" name="WorkbenchSelectorPositionLabel">
+             <property name="text">
+              <string>Workbench selector position :</string>
+             </property>
+            </widget>
+           </item>
+           <item>
+            <widget class="QComboBox" name="WorkbenchSelectorPosition">
+             <property name="toolTip">
+              <string>Customize where the workbench selector appears (restart required).
+
+'Toolbar': In the toolbars, as a movable toolbar.
+'Left Corner': In the menu bar, on the left corner.
+'Right Corner': In the menu bar, on the right corne.</string>
+             </property>
+            </widget>
```

Comment:
Why not use a `PrefComboBox` ? Why not populate the items here ?

Typo in the tooltip : `right corner.`

Reply:
I reused the syntax from the above, treeMode combobox, such that the code is more homogeneous.

Reply:
If it's not the best way to do things, maybe a next PR can change that for all comboboxes.